### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by Johanna1988**

### CODE REVIEW

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package main.java;
+import java.util.Scanner;
+public class Calculator {
+    public static void main(String[] args) {
+        int number1, number2, result;
```
Comment (Maks-Chornyi) [inline src/main/java/Calculator.java]: Declare variables in one line is bad practice. (Read java convention once again just to summarise)
instead `int a,b,s;`
use
int a;
int b;
int c;

File: src/main/java/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package main.java;
+import java.util.Scanner;
+public class Calculator {
+    public static void main(String[] args) {
```
Comment (Maks-Chornyi) [inline src/main/java/Calculator.java:5]: just for better vision provide empty line between class' name and method

File: src/main/java/Calculator.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        System.out.println("Please enter first number");
+        number1 = input.nextInt();
+        System.out.println("Please provide an action: /, *, -, + ");
+        action = input.next().charAt(0);
+        System.out.println("Please enter second number");
+        number2 = input.nextInt();
+        switch (action) {
+            case '/':
+                result = number1 / number2;
+                System.out.println("The result is:");
```
Comment (Maks-Chornyi) [inline src/main/java/Calculator.java]: Good practice to add space after semicolon "The result is: "

File: src/main/java/Calculator.java (hunk 1 above)
Comment (Johanna1988) [reply src/main/java/Calculator.java]: changed

File: src/main/java/Calculator.java (hunk 2 above)
Comment (Johanna1988) [reply src/main/java/Calculator.java:5]: added empty line

File: src/main/java/Calculator.java (hunk 3 above)
Comment (Johanna1988) [reply src/main/java/Calculator.java]: done

**Review by Maks-Chornyi — APPROVED** [review]

**Pull request merged**